### ISSUE TITLE
A small error in backward propagation

### ISSUE THREAD

**Issue by MercuriXito** [genforce/genforce#8]
Thanks for the release of this awesome repo and related pre-trained models, and recently I found a small error while using the pre-trained models.

The following line would incur non-contiguous tensor in network forwarding for using the different parts of `style_split`. So it might cause `Runtime Error: non-contiguous input` in backward propagation.

https://github.com/genforce/genforce/blob/e7e82e47027f8653636b379a2d314d8e4ca91ef6/models/stylegan_generator.py#L658

And I suggest adding `x = x.contiguous()` would help.

**Comment by justimyhxu** [comment]
Hi,
We do not meet the error. Can you provide your environment and the message for the error in detail?
We recommend using PyTorch 1.5+ for this codebase.

**Comment by MercuriXito** [comment]
Hi, Thanks for the early reply!
Yes, I'm using PyTorch 1.4.0 with cuda10.1, cudnn 7.6.5. The error is simple, when I try to back prop through the network, here comes out the following message:

```
RuntimeError: cuDNN error: CUDNN_STATUS_NOT_SUPPORTED. This error may appear if you passed in a non-contiguous input.
```

I simply check if any ouput of the layers turned to be non-contiguous, so I found the line that mentioned before. The output of that line happened to be non-contiguous. Things go well if I turn it to contiguous. 

And I tested it with PyTorch-1.5, which turned out to work well. Thanks for the clarification.
